xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-dialog/modal.js

```diff
@@ -49,7 +49,8 @@ define('modal', ['jquery', 'l10n!modal', 'bootstrap'], function($, translations)
       title: '',
       content: '',
       acceptLabel: translations.get('ok'),
-      dismissLabel: translations.get('cancel')
+      dismissLabel: translations.get('cancel'),
+      form: false
```

Comment:
Some doc of what `form` does, and when it should be used, here and in xwiki.org would be nice.

Reply:
Added a comment about it in https://github.com/xwiki/xwiki-platform/pull/2970/commits/90cd576e17180c0707650aaccfd087d274daba85 .
I'll provide some documentation and a small release note for devs when we merge the PR :ok_hand: